ci/pipeline

```diff
@@ -221,3 +235,34 @@ def jenkins(): Unit = {
   if(utils.isPullRequest) pr()
   else master()
 }
+
+/**
+ * Executes the Community Release which includes:
+ *  1. tarball with version details
+ *  2. release of docker with that version
+ *  3. git tag of version (pushed to server)
+ *
+ * @return Version and artifact description of Marathon build.
+ */
+@main
+def release(releaseVersion: String, gitSha: String, latest: String = "False"): Unit = {
+  println(s"Releasing version: $releaseVersion ")
+  %('git, "checkout", gitSha)
+  %('git, "tag", "-a", releaseVersion, "-m", releaseVersion)
+  val version = build()
+
+  // Uploads
+  uploadTarballPackagesToS3(version, s"releases/$releaseVersion")
+  // push docker
```

Comment:
Nit: I don't think this comment is required since the command actually says `docker push` 🙂.